docs/Using-Fleet/REST-API-v1.md

```diff
@@ -4671,16 +4671,16 @@ None.
     }
   },
   "update_interval": {
-    "osquery_detail": 3600000000000,
-    "osquery_policy": 3600000000000
+    "osquery_detail": "1h0m0s",
```

Comment:
This is a breaking api change. Are we okay with that?

Reply:
https://github.com/fleetdm/fleet/pull/5123#discussion_r849819635